plugins/pixart-rf/fu-pxi-ble-device.c

```diff
@@ -787,6 +883,7 @@ fu_pxi_ble_device_get_model_info(FuPxiBleDevice *self, GError **error)
 			    sizeof(model_name),
 			    error))
 		return FALSE;
+	g_debug("model name%s",model_name);
```

Comment:
not sure this debugging is useful -- if it is then it should be in the `_to_string()` output too.